htlcswitch/link.go

```diff
@@ -2344,6 +2344,9 @@ func (l *channelLink) canSendHtlc(policy ForwardingPolicy,
 
 	// Check to see if there is enough balance in this channel.
 	if amt > l.Bandwidth() {
+		l.log.Errorf("insufficient bandwidth to route htlc on "+
```

Comment:
We already [prefix](https://github.com/lightningnetwork/lnd/blob/master/htlcswitch/link.go#L407) the link's logger with short channel id so you can just log amount + bandwidth here, otherwise channel id is repeated.